exchanges/orderbook/orderbook_types.go

```diff
@@ -79,7 +79,7 @@ type Level struct {
 	OrderCount        int64
 }
 
-// Book contains a moment in time snapshot from an Depth
+// Book stores an orderbook
```

Comment:
🦍 👑  Just aside, but `contains` felt better, and `store` feels reserved.
```suggestion
// Book contains an orderbook
```